app/Http/Controllers/Admin/AuthController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Auth;
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Requests\Admin\LoginRequest;
+
+class AuthController extends AdminBaseController
+{
+
+    public function __construct() {
+  		$this->middleware('guest', ['except' => 'getLogout']);
+  	}
+    /**
+     * [show page lofin for admin]
+     * @return Response
+     */
+    public function getLogin()
+    {
+      return view('admin.login');
```

Comment:
The view path is not right, should be `module.controller.action`. It should be `admin.auth.login`